toolkit/Makefile

```diff
@@ -141,6 +141,10 @@ include $(SCRIPTS_DIR)/utils.mk
 
 ######## REMAINING BUILD FLAGS ########
 
+ifeq ($(RUN_CHECK),y)
+	override PACKAGE_URL_LIST   += https://packages.microsoft.com/azurelinux/$(RELEASE_MAJOR_ID)/prod/extended/$(build_arch)
```

Comment:
Was this actually needed? `PACKAGE_URL_LIST` in theory should only be needed for generating a toolchain. I would think adding to the REPO_LIST would be better.

Reply:
Could you explain in more detail what is the goal of this change? Since it doesn't influence regular package builds, I'm not sure what we're trying to do.